app/javascript/dashboard/routes/dashboard/Dashboard.vue

```diff
@@ -136,25 +121,32 @@ export default {
 </script>
 
 <template>
-  <div class="flex flex-wrap app-wrapper text-n-slate-12">
+  <div class="flex flex-grow overflow-hidden text-n-slate-12">
     <NextSidebar
+      :is-mobile-sidebar-open="isMobileSidebarOpen"
       @toggle-account-modal="toggleAccountModal"
       @open-key-shortcut-modal="toggleKeyShortcutModal"
       @close-key-shortcut-modal="closeKeyShortcutModal"
       @show-create-account-modal="openCreateAccountModal"
+      @close-mobile-sidebar="closeMobileSidebar"
     />
-    <main class="flex flex-1 h-full min-h-0 px-0 overflow-hidden">
-      <UpgradePage
-        v-show="showUpgradePage"
-        ref="upgradePageRef"
-        :bypass-upgrade-page="bypassUpgradePage"
-      />
+
+    <main class="flex flex-1 h-full w-full min-h-0 px-0 overflow-hidden">
       <template v-if="!showUpgradePage">
         <router-view />
         <CommandBar />
         <CopilotLauncher />
+        <MobileSidebarLauncher
+          :is-mobile-sidebar-open="isMobileSidebarOpen"
+          @toggle="toggleMobileSidebar"
+        />
         <CopilotContainer />
       </template>
+      <UpgradePage
+        v-else
+        ref="upgradePageRef"
+        :bypass-upgrade-page="bypassUpgradePage"
+      />
```

Comment:
These changes will fix the console warnings shown in the screenshot below.
<img width="569" height="208" alt="image" src="https://github.com/user-attachments/assets/d87026fd-895f-4c00-b2c2-839201552a71" />